src/windows/main_window.py

```diff
@@ -2225,7 +2323,7 @@ def SetWindowTitle(self, profile=None):
                     save_indicator,
                     _("Untitled Project"),
                     profile,
-                    "OpenShot Video Editor",
+                    "Julk's OpenShot Video Editor",
```

Comment:
:thinking:

Reply:
I use Julk as my username sometimes so I thought it would be fun to put that there lol